### DISCUSSION TITLE
Prometheus Pod restarting multiple times

### DISCUSSION THREAD

**Discussion by saiben10** [prometheus/prometheus#14967] [Q&A]
### What did you do?

Prometheus server getting error restarting pod irregularly
 
2024-09-04T12:33:48.0863293217
 
ts-2024-09-04T12:33:48.0852
 
caller-query_logger.go:71 level-error
 
component-activeQueryTracker msg-"Failed to read
 
query log file" err="read/data/[queries.active](http://queries.active/): input/ output error" 2024-09-04T12:33:48.0863554222
 
ts-2024-09-04T12:33:48.0862
 
caller-query_logger.go:90 level-error
 
component-activeQueryTracker Deployed in kubernetes using nfs share whenever we redeploying the pod again comes into running state what might be the issue we faced multiple times we installed in on aks cluster which is perfectly working in other cluster
 
 
This is prod please help me on this urgently need
has context menu

### What did you see instead? Under which circumstances?

Not using anything how to overcome this

**Comment by machine424** [comment]
Please take a look at the `CAUTION` block at https://prometheus.io/docs/prometheus/latest/storage/#operational-aspects, also Prometheus emits a log about NFS FS not being supported.
I'll convert this into a discussion instead someone from the community could help.

**Reply by saiben10** [reply]
we are not using nfs share we are using azure file disks here
